src/js/tracker.js

```diff
@@ -2052,9 +2038,33 @@
 		apiMethods.enableActivityTracking = function (minimumVisitLength, heartBeatDelay) {
 			if (minimumVisitLength === parseInt(minimumVisitLength, 10) &&
 				heartBeatDelay === parseInt(heartBeatDelay, 10)) {
-				activityTrackingEnabled = true;
-				configMinimumVisitTime = new Date().getTime() + minimumVisitLength * 1000;
-				configHeartBeatTimer = heartBeatDelay * 1000;
+				activityTrackingConfig = {
+					activityTrackingEnabled: true,
+					configMinimumVisitLength: minimumVisitLength,
+					configHeartBeatTimer: heartBeatDelay * 1000
+				}
+			} else {
+				helpers.warn("Activity tracking not enabled, please provide integer values " +
+					"for minimumVisitLength and heartBeatDelay.")
```

Comment:
And this string appears twice.  It depends how much you care about minification, of course!

Reply:
Opened #791 to sort this out for all repeated strings